src/pages/ProductList/Product/Product.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import Tag from './Tag';
+import './Product.scss';
+import { Link } from 'react-router-dom';
```

Comment:
```suggestion
import React from 'react';
import { Link } from 'react-router-dom';
import Tag from './Tag';
import './Product.scss';
```
import 순서 컨벤션 지켜주세요!